src/main/java/org/wildfly/security/sasl/digest/DigestSaslServer.java

```diff
@@ -319,6 +328,18 @@ public String getAuthorizationID() {
         return authzid;
     }
 
+    /* (non-Javadoc)
+     * @see javax.security.sasl.SaslServer#getNegotiatedProperty(String)
+     */
+    @Override
+    public Object getNegotiatedProperty(final String propName) {
+        assertComplete();
+        if (Sasl.BOUND_SERVER_NAME.equals(propName)) {
+            return boundServerName;
+        }
+        return null;
```

Comment:
super.getNegotiatedProperty(propName) should be returned here instead of null

Reply:
Fixed.